bazel_integration_test/script_test.bzl

```
@@ -0,0 +1,91 @@
+load(
+    "//bazel_integration_test:defs.bzl",
+    "bazel_integration_test",
+)
+
+_RUNNER_SCRIPT = """
+#!/bin/bash
+
+script="%s"
+bazel="${BIT_BAZEL_BINARY:-}"
+
+[[ -n "${bazel:-}" ]] || exit_with_msg "Must specify the location of the Bazel binary."
+
+working_dir="."
```

```suggestion
working_dir="${BIT_WORKSPACE_DIR:-}"
```
We pass along the workspace directory as an env variable. This will ensure that the script runs in that directory. Is that the intent?

---

That assumes a workspace directory already exists in the source tree, and so isn't appropriate for this. The correct location is the execution point in the test tmpdir.